src/rt/model.c

```diff
@@ -1754,28 +1768,41 @@ static void reset_coverage(rt_model_t *m)
 {
    assert(m->cover == NULL);
 
-   if ((m->cover = cover_read_tags(m->top)) == NULL)
+   fbuf_t *f = cover_open_lib_file(m->top, FBUF_IN, false);
+   if (f == NULL)
       return;
 
-   int32_t n_stmts, n_conds;
-   cover_count_tags(m->cover, &n_stmts, &n_conds);
+   m->cover = cover_read_tags(f);
+
+   int32_t n_stmts, n_branches, n_toggles;
+   cover_count_tags(m->cover, &n_stmts, &n_branches, &n_toggles);
 
    int32_t *cover_stmts = jit_find_symbol(m->jit, ident_new("cover_stmts"));
    if (cover_stmts != NULL)
       memset(cover_stmts, '\0', sizeof(int32_t) * n_stmts);
 
-   int32_t *cover_conds = jit_find_symbol(m->jit, ident_new("cover_conds"));
-   if (cover_conds != NULL)
-      memset(cover_conds, '\0', sizeof(int32_t) * n_conds);
+   int32_t *cover_branches = jit_find_symbol(m->jit, ident_new("cover_branches"));
```

Comment:
Rebase and replace this with `ffi_find_symbol`.

Reply:
Done